fiftyone/brain/internal/core/pinecone.py

```diff
@@ -183,17 +191,29 @@ class PineconeSimilarityIndex(SimilarityIndex):
     def __init__(self, samples, config, brain_key, backend=None):
         super().__init__(samples, config, brain_key, backend=backend)
         self._index = None
+        self._pinecone = None
         self._initialize()
 
     def _initialize(self):
-        pinecone.init(
-            api_key=self.config.api_key,
-            environment=self.config.environment,
-            project_name=self.config.project_name,
-        )
+        try:
```

Comment:
I believe that `pinecone.init()` was removed in pinecone v3. That's multiple versions ago, so after a bit more thought, I think it would be best to just require:

```py
    def ensure_usage_requirements(self):
        fou.ensure_package("pinecone-client>=3")
```

so that we can remove the `if` statements in this implementation and keep it clean. Sound good to you?

Reply:
Please double check that `pinecone-client>=3` is in fact correct 😄

Reply:
Done. Anything >= 3.2 should be fine. 

`pinecone-client==3.0` is a mystery as it neither has `init` not `Pinecone` :shrug: